### PR TITLE
src/rgw/rgw_admin: show -k option

### PR DESCRIPTION
**Pull request by dorindabassey**
added the ceph switch -k option
for the radosgw-admin --help command to also show the -k option

Fixes: https://tracker.ceph.com/issues/12423

Signed-off-by: dorindabassey <dorindabassey@gmail.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/rgw/rgw_admin.cc [hunk 1]
```diff
@@ -300,6 +300,7 @@ void usage()
   cout << "   --access=<access>         Set access permissions for sub-user, should be one\n";
   cout << "                             of read, write, readwrite, full\n";
   cout << "   --display-name=<name>     user's display name\n";
+  cout << "    -k		        manually specify a key file\n";
```
Comment (yuvalif) [inline src/rgw/rgw_admin.cc:303]: please align the text

Comment (dorindabassey) [reply src/rgw/rgw_admin.cc]: Alright, Done

**Comment by yuvalif** [comment]
@dorindabassey it seems like the original ticket is obsolete. could not find the `-k` option in the file.
sorry about that.

**Comment by dorindabassey** [comment]
Alright, it's ok thanks.

**Pull request closed without merging**